sdks/go/pkg/beam/io/xlang/kafkaio/kafka.go

```diff
@@ -92,6 +97,9 @@ const (
 // Read requires the address for an expansion service for Kafka Read transforms,
 // a comma-seperated list of bootstrap server addresses (see the Kafka property
 // "bootstrap.servers" for details), and at least one topic to read from.
+// If an expansion service address is not provided, one will be automatically
+// started; however this is slower than having a persistent expansion service
+// running.
```

Comment:
Lets be explicit about the empty string here and below with the Write documentation.

```suggestion
// If an expansion service address is provided as "", an appropriate expansion
// service will be automatically started; however this is slower than having a 
// persistent expansion service
// running.
```

We likely want to have alternative functions to call in the future to change them to an optional parameter and can mark these as Deprecated, but lets wait until we have Go IO standards for that cleanup.